api/src/main/java/io/lumeer/api/model/LinkInstance.java

```diff
@@ -19,25 +19,94 @@
 
 package io.lumeer.api.model;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 
-public interface LinkInstance {
+public class LinkInstance {
+
+   public static final String ID = "_ID";
```

Comment:
Why do we define `_id` field here? This class should not be mapped to MongoDB in any way.

Reply:
It is unused. I will remove it.